widgetry/src/mapspace/world.rs

```diff
@@ -118,8 +118,15 @@ pub struct ObjectBuilder<'a, ID: ObjectID> {
 impl<'a, ID: ObjectID> ObjectBuilder<'a, ID> {
     /// Specifies the geometry of the object. Required.
     pub fn hitbox(mut self, polygon: Polygon) -> Self {
-        assert!(self.hitbox.is_none(), "called hitbox twice");
-        self.hitbox = Some(polygon);
+        assert!(self.hitboxes.is_empty(), "called hitbox twice");
+        self.hitboxes = vec![polygon];
+        self
+    }
+
+    /// Specifies the geometry of the object as a multipolygon.
```

Comment:
The change here is to let something in a `World` have multiple polygons as the hitbox. If more places in the code wind up needing something like this, then a `MultiPolygon` type becomes useful. :)